package.json

```diff
@@ -46,7 +46,7 @@
         "stylelint": "stylelint \"./**/*.{css,scss,vue}\"",
         "stylelint-fix": "stylelint \"./**/*.{css,scss,vue}\" --fix",
         "svgo": "svgo -r . --disable='removeDimensions,removeViewBox,minifyStyles'",
-        "test": "mocha --recursive",

```

Comment:
let's use mocha and not jest

Reply:
https://vue-test-utils.vuejs.org/guides/testing-single-file-components-with-mocha-webpack.html#setting-up-mochapack